usr/log.c

```diff
@@ -336,6 +336,20 @@ void log_info(const char *fmt, ...)
 	va_end(ap);
 }
 
+void log_session_reopen(int level, iscsi_session_t *session, const char *fmt, ...)
+{
+	int reopen_cnt = session->reopen_cnt;
+	int reopen_log_freq = session->reopen_log_freq;
+
+	if ((log_level >= level) && ((log_level > 2) ||
+		(reopen_cnt % reopen_log_freq) == 0)) {
+			va_list ap;
+			va_start(ap, fmt);
+			log_func(LOG_DEBUG, log_func_priv, fmt, ap);
```

Comment:
but you're still logging at the debug level, even if debug logging level is set to 0 in iscsid?